frontend/src/styles/main.scss

```diff
@@ -6,10 +6,21 @@ $link-color: #0d6efd;
 
 @import "./bootstrap";
 @import "flatpickr/dist/flatpickr.css";
+@import 'swiper/swiper.min.css';
+@import 'swiper/swiper-bundle.css';
```

Comment:
if swiper provides a non-minimized css it'd be better to use it because then it'd be legible in our source maps.   We compress the css ourselves when bundling and doing it twice is extra work